\documentclass[11pt]{article}

\usepackage{fullpage}
\setlength{\parindent}{0in}

% Font set to san-serif
\usepackage[T1]{fontenc}
\usepackage[scaled]{helvet}
\renewcommand*\familydefault{\sfdefault}
\usepackage{sfmath}

% Wingdings for star labels
\usepackage{pifont}
% Reverse numbering of publication lists
%\usepackage{revnum}

\usepackage{longtable}
\setlength{\LTpre}{0pt}
\setlength{\LTpost}{12pt}

\usepackage[explicit]{titlesec}
\titlespacing\section{0pt}{\baselineskip}{0pt}
\titlespacing\subsection{0pt}{\baselineskip}{0pt}
\titleformat{\subsection}{}{\thesubsection}{}{\underline{#1}}
  
\usepackage{fancyhdr}
\pagestyle{fancy}
\renewcommand{\headrulewidth}{0pt}
\usepackage{lastpage}
\cfoot{Curriculum Vitae - \thepage\ of \pageref{LastPage}\\\scriptsize dated \today}

\usepackage{xspace}
\newcommand{\dadi}{dadi\xspace}

\usepackage[colorlinks=False,pdfborder={0 0 0}]{hyperref}
\urlstyle{same}

\begin{document}

\begin{center}
\textbf{\Large Ryan N. Gutenkunst}\\
\url{http://gutengroup.arizona.edu}\\
\end{center}

\section*{Education}
\begin{longtable}[l]{l l}
1998--2002 & California Institute of Technology, Pasadena, CA; B.S. with Honor, Physics\\
2002--2008 & Cornell University, Ithaca, NY; Ph.D., Physics\\
          & Dissertation Title: ``Sloppiness, Modeling, and Evolution in Biochemical Networks''\\
          & Dissertation Advisor: Prof.\ James P. Sethna
\end{longtable}

\section*{Employment}
\begin{longtable}[l]{l l}
2007--2008 & Postdoctoral Fellow, Cornell University\\%, Ithaca, NY\\
          & Mentors: Prof.\ Scott H. Williamson and Prof.\ Carlos D. Bustamante\\
2009--2010 & Postdoctoral Fellow, Los Alamos National Laboratory\\%, Los Alamos, NM\\
          & Mentor: Dr.\ Byron Goldstein\\
2010--2017 & Assistant Professor, Dept of Molecular \& Cellular Biology, University of Arizona\\
%& Tucson, AZ\\
2017--2023 & Associate Professor with Tenure\\
2023--present & Professor with Tenure\\
& Affiliations: Departments of Ecology \& Evolutionary Biology and Epidemiology \&\\
&Biostatistics; Graduate Interdisciplinary Programs (GIDPs) in Applied BioSciences,\\
&Applied Mathematics, Cancer Biology, Data Science \& Statistics, and Genetics;\\
&BIO5~Institute\\
2017--2023 & Associate Department Head\\
2023 & Interim Department Head\\
2024--present & Department Head\\
\end{longtable}

\section*{Honors and Awards}
\begin{longtable}[l]{l l}
1998--2002 & National Merit Scholar, American Standard Corporation\\
2000, 2001      & Summer Undergraduate Research Fellowship, California Institute of Technology\\
2002--2004 & NSF Integrative Graduate Education and Research Traineeship (IGERT) in Nonlinear\\
                  & Systems, Cornell University\\
2004      & Honorable Mention, NSF Graduate Research Fellowship\\
2004--2006 & NIH Molecular Biophysics Training Grant, Cornell University\\
2009--2010 & Center for Nonlinear Studies postdoctoral appointment, Los Alamos National Lab\\
2009 & 3rd place, District 23 Toastmasters Humorous Speech Contest\\
%2013 & Distinguished Early-Career Teaching Award, College of Science, University of Arizona\\
2014 & Kavli Fellow, US National Academy of Sciences and The Kavli Foundation\\
2021 & Finisher, Ironman Arizona\\ %, time of 13:43:01\\
\end{longtable}

\section*{Service/Outreach}
\subsection*{Local Outreach}
\begin{longtable}[l]{l l}
2013 & Leader, Tucson Festival of Books: Book club on Armand Leroi's ``Mutants''\\
2016 & Pen Pal, Laffer Middle School science students\\
2016 & Speaker, \href{https://www.youtube.com/watch?v=0s-kFTu24dg}{UA Science Cafe: ``Genetic Engineering: From Jurassic Park to Gattaca``}\\
%2019 & Skype-a-Scientist with 7th grade and high school students\\
2019 & Discussion with Tucson Hard Sci-Fi Writers \& Artists Galactic Cabal\\
2020 & Speaker, \href{https://www.youtube.com/watch?v=je5GcyU6CYc}{UA Science Cafe: ``Genetic Testing: Hype versus Reality''}\\
2023-present & Host, high school researchers in my group (4) % Hannah Mendoza, Ashwin Parasanna, Auhonoa Shil, Abhiram Chervu
\end{longtable}

\subsection*{National Outreach}
\begin{longtable}[l]{l l}
2018--2021 & Skype-a-Scientist with K-12 students\\
\end{longtable}

\subsection*{Departmental Service}
\begin{longtable}[l]{l l}
2010--2013 & Co-Chair, MCB/CMM/CBC/IMB joint departmental retreat committee\\
%2012-present & Chair, MCB BioMath curriculum committee\\
%2012-present & Member, MCB faculty search committee\\
2012--present & Member, Undergraduate curriculum committee\\
2012--2013 & Chair, Website committee\\
2013--2015 & Member, Website committee\\
2012, 2016, 2017, 2021 & Member, Faculty peer evaluation committee\\
2012 & Member, Faculty search committee\\
2015--2017 & Member, Discretionary fund allocation committee\\
2016 & Member, Astrobiology faculty search committee\\
2017 & Member, Cancer bioinformatics search committee\\
2017--2023 & Director of Graduate Studies, MCB Accelerated Master's Program\\
2018 & Co-chair, MCB Academic Program Review committee\\
2018--present & Faculty mentor for Prof.\ Megha Padi\\
2019--2021 & Faculty mentor for Prof.\ Bet{\"u}l Ka{\c c}ar\\
2021 & Member, Department Head search committee\\
2022 & Member, Disretionary fund allocation committee\\
2022 & Member, Professor of Practice hiring committee\\
2022--2023 & Director, Undergraduate and Master's Education
\end{longtable}

\subsection*{College/University Service}
\begin{longtable}[l]{l l}
2012 & Internal reviewer, Packard Fellowships for Science and Engineering\\
2012--2016 & Mentor for Arizona Assurance and Arizona Science, Engineering, and Mathematics\\
                        & Scholars\\
2013 & Internal reviewer, Blavatnik Awards for Young Scientists\\
2014 & Member, MCB/CBC/EEB/CS joint faculty search committee\\
2014 & Member, Statistics GIDP graduate advisory committee\\
2017 & Internal reviewer, Basic/Clinical Partnerships seed grant, UA Cancer Center\\
2017--present & Member, Steering Committee for NIH T32 training grant ``Computational and\\
&mathematical modeling of biomedical systems''\\
2017--2023 & Director of Graduate Studies, MCB track of Professional Science Master's in \\
& Applied Biosciences GIDP\\
2018, 2019 & Internal reviewer, Research, Discovery \& Innovation grants\\% (12 total)\\
2019, 2020 & Internal reviewer, Astronaut Scholarship\\
%2019 & Internal reviewer, Research, Discovery \& Innovation grants (5)\\
%2020 & Member, Master's in Genetics curriculum committee\\
2020--present & Member, Executive Committee for Genetics GIDP\\
2022 & Chair, College of Science committee for resolution of grade appeal\\
2022--present & Member, Design Committee for Applied Statistics and Data Science\\ & Professional Science Master's Degree\\
2023--present & Faculty Advisor, UA TriCats triathlon club team\\
2023 & Internal reviewer, CNRS-UArizona Graduate Research Fellowship \& International Mobility\\
2023 & Co-Organizer, Arizona-Los Alamos Days meeting
\end{longtable}

\subsection*{National/International Service}
\begin{longtable}[l]{l l}
%2009--2017 & Lecturer, q-bio Summer School on Cellular Information Processing\\
%2009 - present & Lecturer, \href{http://q-bio.org/wiki/The_Third_q-bio_Summer_School_on_Cellular_Information_Processing}{Third q-bio Summer School on Cellular Information Processing}\\
%2010 & Co-Organizer, \href{http://q-bio.org/wiki/The_Fourth_q-bio_Summer_School_on_Cellular_Information_Processing}{Fourth q-bio Summer School on Cellular Information Processing}\\
2010 & Panelist, National Science Foundation\\% (Graduate Research Fellowship Program)\\
%2011 & Lecturer, \href{http://q-bio.org/wiki/The_Fifth_q-bio_Conference_on_Cellular_Information_Processing}{Fifth q-bio Summer School on Cellular Information Processing}\\
%2011-present & Review Editor, Frontiers in Evolutionary and Population Genetics\\
2013 & Panelist, National Science Foundation\\% (Graduate Research Fellowship Program)\\
2013 & Co-Organizer, \href{http://www.birs.ca/events/2013/5-day-workshops/13w5080}{Banff International Research Station workshop on ``Mathematical}\\
&\href{http://www.birs.ca/events/2013/5-day-workshops/13w5080}{Tools for Evolutionary Systems Biology''}\\
%2013 & Panelist, National Science Foundation\\%Graduate Research Fellowship Program\\
%2013 & Lecturer, \href{http://q-bio.org/wiki/The_Seventh_q-bio_Summer_School}{Seventh q-bio Summer School on Cellular Information Processing}\\ % Textbook chapter reviews for Garland Science and W.H. Freeman and Company.
2014 & Co-Organizer, symposium on ``Evolutionary Systems Biology of Networks'' at the\\
&Annual Meeting of the Society for Molecular Biology and Evolution\\
%2015 & Lecturer, \href{http://q-bio.org/wiki/The_Ninth_q-bio_Summer_School}{Ninth q-bio Summer School on Cellular Information Processing}\\
2015--present & Associate Editor, BMC Ecology and Evolution (previously BMC Evolutionary Biology)\\
%2016 & Lecturer, \href{http://evomics.org/workshops/workshop-on-molecular-evolution/2016-workshop-on-population-and-speciation-genomics-cesky-krumlov/}{Workshop on Population and Speciation Genomics}, \u{C}esk\'{y} Krumlov,\\
%&Czech Republic\\
2017 & Panelist, National Science Foundation\\% (Graduate Research Fellowship Program)\\
2018 & Co-organizer, workshop on Population Genomics Simulation at Cold Spring\\
& Harbor Laboratory\\
2019, 2021 & Organizer, Arizona Population Genetics Group workshop\\
%2017--present & External reference for tenure cases (1)
2020 & Panelist, National Science Foundation\\% (Graduate Research Fellowship Program)\\
2020 & Lecturer, SACNAS meeting on NSF GRFP application, University of Houston\\
2021 & Panelist, National Science Foundation\\% (Graduate Research Fellowship Program)\\
2021 & Panelist, National Science Foundation\\% (Rules of Life: Emergent Networks)\\
2023 & Supervisor, SMBE Virtual Lab Meeting program\\
2023 & Organizer, PopSim satellite meeting, Cold Spring Harbor Laboratory\\
2023 & Organizing Committee, Population, Evolutionary, and Quantitative Genetics conference\\
\end{longtable}

\subsection*{Ad-hoc Journal Reviewer}
%\underline{Prior to Associate Professor:}
{Science~(1)},
{Cell~(3)},
{Proceedings of the National Academy of Sciences USA~(2)},
{eLife~(1)},
{Molecular Biology and Evolution~(8)},
{Genome Research~(2)},
{PLoS Genetics~(4)},
{PLoS Computational Biology~(3)},
{Molecular Systems Biology~(2)},
{Molecular Ecology~(4)},
{Genetics~(14)}, 
{Evolution~(3)}, 
{Proceedings of the Royal Society B~(1)},
{Molecular Cancer Research~(1)},
{Bioinformatics~(2)},
{Molecular Ecology Resources~(1)},
{G3: Genes, Genomes, Genetics~(1)},
{Journal of Molecular Evolution~(1)},
{IET Systems Biology~(1)}, 
{BMC Bioinformatics~(1)},
{BMC Systems Biology~(3)}, 
{BMC Evolutionary Biology~(3)},
{GigaScience~(3)},
{Theoretical Population Biology~(2)},
{Molecular BioSystems~(1)},
{Bulletin of Mathematical Biology~(1)},
{Journal of Computational and Graphical Statistics~(1)},
%{Physical Review E},
{Physical Biology~(1)},
{Philosophical Transactions of the Royal Society B~(1)},
{PLoS ONE~(5)},
{Interface Focus~(1)},
{Frontiers in Evolutionary and Population Genetics~(2)},
{Life Sciences~(1)},
{Biotechnology Progress~(1)},
{Garland Science~(1)},
{W.H. Freeman and Company~(1)}

\subsection*{Ad-hoc Proposal Reviewer}
National Science Foundation (US, 3), National Environmental Research Council (UK, 1), Natural Sciences and Engineering Research Council (Canada, 1), Estonian Research Council (5), Vienna Science and Technology Fund (1),
Wellcome Trust (UK, 2), Human Frontiers Science Program (International, 1), Biotechnology and Biological Sciences Research Council (UK, 1), Israel Science Foundation (2), United States-Israel Binational Science Foundation (1), University of Missouri Research Board~(1), Nevada IDeA Network of Biomedical Research Excellence (1)

\subsection*{External Thesis Reviewer}
McGill University (Canada, 1)

\subsection*{Promotion \& Tenure Package Reviewer}
%Northwestern University (2)
%University of California - Los Angeles (1)
%University of Nevada - Reno (1)
% Mississippi State University (1)
% Northern Arizona U (1)
%Haverford College (1)
7 (Years and institutions not listed to protect confidentiality)


\newenvironment{hanglist}[1][\parindent]{%
    \begin{list}{}{%
        \setlength{\leftmargin}{#1}
        \setlength{\labelwidth}{0pt}
        \setlength{\labelsep}{0pt}
        \setlength{\itemindent}{-#1}
        \setlength{\topsep}{0pt}
        \setlength{\partopsep}{0pt}
        \setlength{\parsep}{0pt}
        \setlength{\itemsep}{0pt}}
    }{%
        \end{list}
    }

    
\newcommand{\trainee}{$^\mathrm{O}$}
\newcommand{\corresponding}{$^\mathrm{C}$\xspace}
\newcommand{\equal}{$^\mathrm{E}$}
\newcommand{\grad}{$^*$}

\section*{Publications/Creative Activity}
Key: \trainee-Gutenkunst group trainee, \corresponding-corresponding author, \equal-contributed equally,  \grad-substantially based on work done as a graduate student.\\
In the scientific fields to which I've contributed most, the authors with the greatest roles in carrying out the work are listed first, and the authors with the greatest roles in supervision are listed last. Conventions for corresponding author vary.

\subsection*{Refereed Journal Articles}
\begin{enumerate}


\item \href{http://doi.org/10.1119/1.1531578}{Black ED, \textbf{Gutenkunst RN} (2003) {An introduction to signal extraction in
  interferometric gravitational wave detectors}.
\newblock \emph{American Journal of Physics} 71:365.}

\item \href{http://doi.org/10.1049/iet-syb:20060065}{\grad Casey FP\corresponding, Baird D, Feng Q, \textbf{Gutenkunst RN}, Waterfall JJ, Myers CR, Brown KS, Cerione RA, Sethna JP (2007) {Optimal experimental design in an epidermal growth factor receptor signalling and down-regulation model}.
\newblock \emph{IET Systems Biology} 1:190.}

\item \href{http://doi.org/10.1196/annals.1407.003}{\grad \textbf{Gutenkunst RN}\corresponding, Casey FP, Waterfall JJ, Myers CR, Sethna JP (2007)
  {Extracting falsifiable predictions from sloppy models.}
\newblock \emph{Annals of the New York Academy of Sciences} 1115:203.}

\item \href{http://doi.org/10.1016/j.jtbi.2006.10.014}{\grad \textbf{Gutenkunst R}\corresponding, Newlands N, Lutcavage M, Edelstein-Keshet L (2007)
  {Inferring resource distributions from Atlantic bluefin tuna movements: an
  analysis based on net displacement and length of track.}
\newblock \emph{Journal of Theoretical Biology} 245:243.}

\item \href{http://doi.org/10.1371/journal.pcbi.0030189}{\grad \textbf{Gutenkunst RN}\corresponding, Waterfall JJ, Casey FP, Brown KS, Myers CR, Sethna JP  (2007) {Universally sloppy parameter sensitivities in systems biology models}.
\newblock \emph{PLoS Computational Biology} 3:e189.}\\
\ding{73}  \href{http://biomedicalcomputationreview.org/content/%E2%80%9Csloppy%E2%80%9D-systems-biology}{Featured in \emph{Biomedical Computation Review}}.\\
\ding{73} ``Exceptional'' evaluation on \textit{Faculty of 1000: Biology}

\item \href{http://doi.org/10.1109/MCSE.2007.60}{\grad Myers CR, \textbf{Gutenkunst RN}, Sethna JP (2007) {Python unleashed on systems
  biology}.
\newblock \emph{Computing in Science \& Engineering} 9:34.}


\item \href{http://doi.org/10.1103/PhysRevE.78.046704}{\grad Casey FP\corresponding, Waterfall JJ, \textbf{Gutenkunst RN}, Myers CR, Sethna JP (2008) {Variational method for estimating the rate of convergence of Markov-chain Monte Carlo algorithms}.
\newblock \emph{Physical Review E} 78:046704.}


\item \href{http://doi.org/10.1016/j.copbio.2008.06.008}{\grad Daniels BC, Chen YJ, Sethna JP\corresponding, \textbf{Gutenkunst RN}, Myers CR (2008) {Sloppiness,
  robustness, and evolvability in systems biology}.
\newblock \emph{Current Opinion in Biotechnology} 19:389.}


\item \href{http://doi.org/10.1093/molbev/msp190}{Andr\'{e}s AM\corresponding, Hubisz MJ, Indap A, Torgerson DG, Degenhardt JD, Boyko AR,
  \textbf{Gutenkunst RN}, White TJ, Green ED, Bustamante CD, Clark AG, Nielsen R (2009)
  {Targets of balancing selection in the human genome}.
\newblock \emph{Molecular Biology and Evolution} 26:2755.}

\item
\href{http://doi.org/10.1101/gr.088898.108}{Auton A, Bryc K, Boyko AR, Lohmueller KE, Novembre J, Reynolds A, Indap A,
  Wright MH, Degenhardt JD, \textbf{Gutenkunst RN}, King KS, Nelson MR, Bustamante CD\corresponding
  (2009) {Global distribution of genomic diversity underscores rich complex
  history of continental human populations.}
\newblock \emph{Genome Research} 19:795.}

\item
\href{http://doi.org/10.1371/journal.pgen.1000695}{\textbf{Gutenkunst RN}\corresponding, Hernandez RD, Williamson SH, Bustamante CD (2009) {Inferring the
  joint demographic history of multiple populations from multidimensional SNP
  frequency data}.
\newblock \emph{PLoS Genetics} 5:e1000695.}

\item
\href{http://doi.org/10.1101/gr.088336.108}{Nielsen R\corresponding, Hubisz MJ, Hellmann I, Torgerson D, Andr\'{e}s AM, Albrechtsen A,
  \textbf{Gutenkunst R}, Adams MD, Cargill M, Boyko A, Indap A, Bustamante CD, Clark AG
  (2009) {Darwinian and demographic forces affecting human protein coding
  genes}.
\newblock \emph{Genome Research} 19:838.}



\item
\href{http://doi.org/10.1186/1471-2105-11-404}{Colvin J, Monine MI, \textbf{Gutenkunst RN}, Hlavacek WS, {Von Hoff} DD, Posner RG\corresponding
  (2010) {RuleMonkey: software for stochastic simulation of rule-based models.}
\newblock \emph{BMC Bioinformatics} 11:404.}
%\begin{hanglist}[0.1in]
%\item 
%- Reports novel software for simulating complex cell signaling models. 
%\item - I added features and fixed bugs in the software described, while using it for another project.
%\end{hanglist}

\item
\href{http://doi.org/10.1038/nature09534}{{The 1000 Genomes Project Consortium} (2010) {A map of human genome variation
  from population-scale sequencing}.
\newblock \emph{Nature} 467:1061.}
%\begin{hanglist}[0.1in]
%\item 
%- Reports the sequencing and analysis of many human genomes.
%\item - I analyzed early data from the project, pointing out flaws in the genomic variant calling that were later corrected.
%\end{hanglist}


\item
\href{http://doi.org/10.1039/C1MB05077J}{Chylek LA, Hu B, Blinov ML, Emonet T, Faeder JR, Goldstein B, \textbf{Gutenkunst RN},
  Haugh JM, Lipniacki T, Posner RG, Yang J, Hlavacek WS\corresponding (2011) {Guidelines for
  visualizing and annotating rule-based models}.
\newblock \emph{Molecular BioSystems} 7:2779.}
%\begin{hanglist}[0.1in]
%\item 
%- Reports a convention for visually representing and annotating models of complex cell signaling systems.
%\item - I contributed to the design of the visualization rules.
%\end{hanglist}

\item
\href{http://doi.org/10.1073/pnas.1019276108}{Gravel S, Henn BM, \textbf{Gutenkunst RN}, Indap AR, Marth GT, Clark AG, Yu F, Gibbs RA,
  {The 1000 Genomes Project}, Bustamante CD\corresponding (2011) {Demographic history and
  rare allele sharing among human populations}.}
\newblock \emph{Proceedings of the National Academy of Sciences USA} 108:11983.\\
%\begin{hanglist}[0.1in]
%\item
%- Reports re-fitting the demographic model from my 2009 \emph{PLoS Genetics} paper to 1000 Genomes Project pilot data and using it to predict the distribution of genetic variants yet to be discovered.
%\item - I extended \dadi to allow analysis of large population samples, and I consulted on the model fitting.
\ding{73} Featured: Wade N.\ (2011) Roots of disease found to vary by continent. \emph{New York Times}.
%\end{hanglist}

\item
\href{http://doi.org/10.1371/journal.pone.0019701}{\textbf{Gutenkunst RN}\corresponding, Coombs D, Starr T, Dustin ML, Goldstein B (2011) {A biophysical
  model of cell adhesion mediated by immunoadhesin drugs and antibodies}.
\newblock \emph{PLoS ONE} 6:e19701.}
%\begin{hanglist}[0.10in]
%\item
%- Reports detailed mathematical modeling of experiments on the drug-mediated adhesion of T cells to specially-prepared surfaces, a scenario designed to mimic the action of these drugs in vivo. Shows that simple models of T cell receptor mobility are inconsistent with the data, suggesting that receptor mobility may be non-trivially controlled by the T cells.
%\item - I developed the model, analyzed the data, and wrote the paper.
%\end{hanglist}

\item 
\href{http://doi.org/10.1038/nature09687}{%
%Locke DP\corresponding, Hillier LW, Warren WC\corresponding, Worley KC, Nazareth LV, Muzny DM, Yang SP,
%  Wang Z, Chinwalla AT, Minx P, Mitreva M, Cook L, Delehaunty KD, Fronick C,
%  Schmidt H, Fulton LA, Fulton RS, Nelson JO, Magrini V, Pohl C, Graves TA,
%  Markovic C, Cree A, Dinh HH, Hume J, Kovar CL, Fowler GR, Lunter G, Meader S,
%  Heger A, Ponting CP, Marques-Bonet T, Alkan C, Chen L, Cheng Z, Kidd JM,
%  Eichler EE, White S, Searle S, Vilella AJ, Chen Y, Flicek P, Ma J, Raney B,
%  Suh B, Burhans R, Herrero J, Haussler D, Faria R, Fernando O, Darr\'{e} F,
%  Farr\'{e} D, Gazave E, Oliva M, Navarro A, Roberto R, Capozzi O, Archidiacono
%  N, {Della Valle} G, Purgato S, Rocchi M, Konkel MK, Walker JA, Ullmer B,
%  Batzer MA, Smit AFA, Hubley R, Casola C, Schrider DR, Hahn MW, Quesada V,
%  Puente XS, Ordo\~{n}ez GR, L\'{o}pez-Ot\'{\i}n C, Vinar T, Brejova B, Ratan
%  A, Harris RS, Miller W, Kosiol C, Lawson HA, Taliwal V, Martins AL, Siepel A,
%  Roychoudhury A, Ma X, Degenhardt J, Bustamante CD, \textbf{Gutenkunst RN}, Mailund T,
%  Dutheil JY, Hobolth A, Schierup MH, Ryder OA, Yoshinaga Y, de~Jong PJ,
%  Weinstock GM, Rogers J, Mardis ER, Gibbs RA, Wilson RK 
The Orangutan Genome Sequencing Consortium
  (2011) {Comparative
  and demographic analysis of orang-utan genomes}.
\newblock \emph{Nature} 469:529.}\\
%\begin{hanglist}[0.10in]
%- Reports the initial high-quality sequencing of the orangutan genome, the resequencing of ten other individuals, and the analysis of the resulting data.
%\item - I designed and carried out a demographic history analysis that revealed a much more recent split time between Bornean and Sumatran orangutans than previously reported. That analysis became Fig. 5 in the final paper and was later validated by an independent modeling approach.
\ding{73} Featured: Stolte D (2011) First analysis of orangutan genome yields surprises. \emph{UANews}.
%\end{hanglist}

\item 
\href{http://doi.org/10.1371/journal.pone.0021747}{Skar H\equal, \textbf{Gutenkunst RN}\equal, {Wilbe Ramsay} K, Alaeus A, Albert J, Leitner T\corresponding (2011)
  {Daily sampling of an HIV-1 patient with slowly progressing disease displays
  persistence of multiple env subpopulations consistent with neutrality}.}
\newblock \emph{PLoS ONE} 6:e21747.
%\begin{hanglist}[0.10in]
%\item
%- Reports the unexpected existence of multiple phylogenetically distinct subpopulations of HIV that persisted over several years in a single patient.
%\item - I designed and carried out an analysis of the subpopulation dynamics that showed them to be consistent with neutral evolution. I also co-wrote the paper.
%\end{hanglist}

\item
\href{http://doi.org/10.1371/journal.pcbi.1001081}{Smith AM, Adler FR, McAuley JL, \textbf{Gutenkunst RN}, Ribeiro RM, McCullers JA,
  Perelson AS\corresponding (2011) {Effect of 1918 PB1-F2 expression on influenza A virus
  infection kinetics}.
\newblock \emph{PLoS Computational Biology} 7:e1001081.}
%\begin{hanglist}[0.1in]
%\item 
%- Reports experiments and corresponding viral kinetic modeling on flu virus with and without a gene present in the 1918 pandemic strain.
%\item - I designed and carried out the uncertainty analysis for the viral kinetics models, which enabled us to distinguish reliable and unreliable predictions and conclusions from the modeling.
%\end{hanglist}



\item
\href{http://doi.org/10.1038/nbt.2050}{Xu X\equal, Liu X\equal, Ge S\equal, Jensen JD\equal, Hu F\equal, Li X\equal, Dong Y\equal, \textbf{Gutenkunst RN}, Fang L, Huang
  L, Li J, He W, Zhang G, Zheng X, Zhang F, Li Y, Yu C, Kristiansen K, Zhang X,
  Wang J, Wright M, McCouch S, Nielsen R\corresponding, Wang J\corresponding, Wang W\corresponding (2012) {Resequencing
  50 accessions of cultivated and wild rice yields markers for identifying
  agronomically important genes.}
\newblock \emph{Nature Biotechnology} 30:105.}
%\begin{hanglist}[0.1in]
%\item 
%- Reports extensive resequencing of rice and corresponding analyses of population structure and natural selection.
%\item - I designed an initial demographic model fit to several of the sequenced rice populations, and I consulted on the population structure analysis.
%\end{hanglist}



\item \href{http://doi.org/10.1371/journal.pone.0077175}{Ma X, Kelley JL, Eilertson K, Musharoff S, Degenhardt JD, Martins AL, Vinar T, Kosiol C, Siepel A, \textbf{Gutenkunst RN}, Bustamante CD\corresponding (2013) Population genomic analysis of ten genomes reveals a rich speciation and demographic history of orang-utans (Pongo pygmaeus and Pongo abelii). \emph{PLoS ONE} 8:e77175.}
%\begin{hanglist}[0.1in]
%\item - Reports additional demographic and selection analyses of orang-utan population resequencing data from the genome paper.
%\item - I designed and carried out the analysis of the distribution of selection coefficients on newly arising mutations. I also co-wrote and edited the manuscript.
%\end{hanglist}


\item
\href{http://doi.org/10.1371/journal.ppat.1003238}{Smith AM\corresponding, Adler FR, Ribeiro RM, \textbf{Gutenkunst RN}, McAuley JL, McCullers JA, Perelson AS (2013) Kinetics of coinfection with influenza A virus and Streptococcus pneumoniae. \emph{PLoS Pathogens} 9:e1003238.}
%\begin{hanglist}[0.1in]
%\item - Reports experiments with mice and corresponding viral kinetics models of simultaneous flu and bacterial infection.
%\item - I designed and carried out the uncertainty and sensitivity analysis that determined which model parameters could be well-constrained by the available data.
%\end{hanglist}


\item \href{http://doi.org/10.1038/ncomms5383}{Holmes WM, Mannakee BK\trainee, \textbf{Gutenkunst RN}, Serio TR\corresponding (2014) Loss of N-terminal acetylation suppresses a prion phenotype by modulating global protein folding. \emph{Nature Communications} 5:4383.}
%\begin{hanglist}[0.1in]
%\item - Reports experiments on Sup35 and proteome-wide bioinformatic analyses that suggest N-terminal acetylation plays an important and general role in protein folding in yeast.
%\item - My Ph.D. student and I designed and carried out the bioinformatic analyses, and we wrote the corresponding section of the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1371/journal.pcbi.1003481}{Jilkine A\trainee, \textbf{Gutenkunst RN}\corresponding (2014) Effect of dedifferentiation on time to mutation acquisition in stem cell-driven cancers. \emph{PLoS Computational Biology} 10:e1003481.}
%\begin{hanglist}[0.1in]
%\item - Reports a novel model for mutation acquisition in cancer that includes the effects of potential dedifferentiation of derived cells into stem cells. Shows that such dedifferentiation can dramatically shorten the time to carcinogenesis, depending on the mechanism of stem-cell homeostasis.
%\item - My postdoc and I developed and analyzed the model, and we co-wrote the paper.
%\end{hanglist}


\item \href{http://doi.org/10.1186/s12862-014-0254-4}{Robinson JD\corresponding, Coffman AJ\trainee, Hickerson MJ, \textbf{Gutenkunst RN} (2014) Sampling strategies for frequency spectrum-based population genomic inference. \emph{BMC Evolutionary Biology} 14:254.}
%\begin{hanglist}[0.1in]
%\item - Reports analysis of the statistical power of \dadi to distinguish between models of demographic history.
%\item - I helped design the analysis and guided Dr.\ Robinson in using \dadi. My programmer carried out much of the computation.
%\end{hanglist}

\item  \href{http://doi.org/10.1093/molbev/msu166}{Veeramah K, \textbf{Gutenkunst RN}, Woerner A, Watkins J, Hammer M\corresponding (2014) Evidence for increased levels of positive and negative selection on the X chromosome versus autosomes in humans. \emph{Molecular Biology and Evolution} 31:2267.}
%\begin{hanglist}[0.1in]
%\item - Reports use of \dadi to infer the distribution of fitness effects of new mutations on both the autosomes and the X chromosome in humans, finding evidence for  faster-X evolution in humans.
%\item - I extended \dadi to model allele frequencies on the X chromosome.
%\end{hanglist}


\item \href{http://doi.org/10.1186/s12862-015-0515-x}{Hermansen RA\equal, Mannakee BK\trainee\equal, Knecht W, Liberles DA\corresponding, \textbf{Gutenkunst RN}\corresponding (2015) Characterizing selective pressures on the pathway for de novo biosynthesis of pyrimidines in yeast. \emph{BMC Evolutionary Biology} 15:232.}
%\begin{hanglist}[0.1in]
%\item - Reports a combined modeling and phylogenetic analysis of an essential yeast metabolic pathway. Show that enzymes with greater influence on the steady-state pathway flux are evolving more slowly. 
%\item - Dr.\ Liberles and I designed the project. My Ph.D. student and I developed the kinetic model of the pathway, and the Liberles lab carried out the phylogenetic analysis. I compared the kinetic and phylogenetic results to reach our conclusions. All collaborators contributed to writing the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1093/molbev/msu284}{Pandya S\trainee, Struck TJ\trainee, Mannakee BK\trainee, Paniscus M\trainee, \textbf{Gutenkunst RN}\corresponding (2015) Testing whether metazoan tyrosine loss was driven by selection against promiscuous phosphorylation. \emph{Molecular Biology and Evolution} 32:144.}
%\begin{hanglist}[0.1in]
%\item - Reports biophysical and population-genetic tests of the hypothesis that selection against promiscuous tyrosine phosphorylation has driven eukaryotes with more tyrosine kinases in their proteome to reduce tyrosine usage in their proteins. Tests four predictions of this hypothesis and finds that none of them hold, casting doubt on the hypothesis.
%\item - My graduate and undergraduate students carried out the analysis, which I designed and supervised. I wrote the paper.
%\end{hanglist}


\item \href{http://doi.org/10.1093/molbev/msv255}{Coffman AJ\trainee, Hsieh P\trainee, Gravel S, \textbf{Gutenkunst RN}\corresponding (2016) Computationally efficient composite likelihood statistics for demographic inference. \emph{Molecular Biology and Evolution} 33:591.}
%\begin{hanglist}[0.1in]
%\item - Reports the application of composite likelihood theory to estimate parameter confidence intervals and perform likelihood ratio tests in the population genetics methods \dadi and \textsc{TRACTS}. Applying this theory dramatically lowers the computational costs of careful modeling, making the methods more useful for researchers without access to high performance computing.
%\item - My programmer carried out most of the analysis, which I designed and supervised. We co-wrote the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1002/ece3.1865}{Edwards T\corresponding, Tollis M, Hsieh P\trainee,  \textbf{Gutenkunst RN}, Liu Z, Kusumi K, Culver M, Murphy RW (2016) Assessing models of speciation under different biogeographic scenarios; an empirical study using multi-locus and RNA-seq analyses. \emph{Ecology and Evolution} 6:379.}
%\begin{hanglist}[0.1in]
%\item - Reports analysis of sequence data from three populations of desert tortoises. Shows that they are genetically isolated from each other and thus well-defined species.
%\item - My Ph.D. student and I designed and carried out the \dadi analysis that revealed the lack of gene flow, and we wrote the relevant sections of the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1101/gr.192971.115}{Hsieh P\trainee, Veeramah KR, Lachance J, Tishkoff SA, Wall JD, Hammer MF\corresponding, \textbf{Gutenkunst RN}\corresponding
(2016) Whole-genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection. \emph{Genome Research} 26:279.}\\
\ding{73}  \href{http://tucson.com/news/local/pygmy-research-reveals-genetic-changes-for-new-environments/article_013dfd0b-d443-5a41-a817-cb4802c9617d.html}{Featured in the \emph{Arizona Daily Star}}.
%\begin{hanglist}[0.1in]
%\item - Reports demographic modeling with \dadi and a scan for signatures of selection, using whole-genome sequencing data from two African Pygmy populations and one African farming population. An important methodological advance is that we reduce false positives by using a null model consisting of genome-scale coalescent simulations that incorporate local mutation and recombination rates. A particularly interesting result is genetic evidence for natural selection on a muscle development gene, which may be related to known differences in muscle physiology and tree-climbing ability between Pygmies and farmers.
%\item - The data were generated in the Hammer and Tishkoff labs. Prof.\ Hammer and I designed and supervised the analysis. My Ph.D. student and I wrote the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1101/gr.196634.115}{Hsieh P\trainee, Woerner AE, Wall JD, Lachance J, Tishkoff SA, \textbf{Gutenkunst RN}, Hammer MF\corresponding
(2016) Model-based analyses of whole genome data reveal a complex evolutionary history involving archaic introgression in Central African Pygmies. \emph{Genome Research} 26:291.}
%\begin{hanglist}[0.1in]
%\item - Builds upon our prior demographic modeling in two African Pygmy populations to identify regions of the genome likely to have been introgressed from extinct hominids. Using these regions, we then tested and rejected the hypothesis of a single introgression event, suggesting that interbreeding between the ancestors of modern humans and archaic hominids was common in human evolution within Africa.
%\item - The data were generated in the Hammer and Tishkoff labs. Prof.\ Hammer and I designed and supervised the analysis. My Ph.D. student, Prof. Hammer, and I wrote the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1371/journal.pgen.1006132}{Mannakee BK\trainee, \textbf{Gutenkunst RN}\corresponding (2016) Selection on network dynamics drives differential rates of protein domain evolution. \emph{PLoS Genetics} 12:e1006132}.
%\begin{hanglist}[0.1in]
%\item - Reports a combined analysis of detailed dynamical models of protein networks and comparative genomic data, showing that network dynamics constrain protein evolution. This contradicts recent knock-out studies arguing that the specific function of a protein only weakly constrains its rate of evolution. Our results suggest that knock-out experiments have been misleading, because they are a poor proxy for evolutionarily-relevant changes.
%\item - My Ph.D. student carried out analysis, which I designed and supervised. We co-wrote the paper.
%\end{hanglist}

\item \href{http://doi.org/10.1534/genetics.115.184812}{Ragsdale AP\trainee, Coffman AJ\trainee, Hsieh P\trainee, Struck TJ\trainee, \textbf{Gutenkunst RN}\corresponding (2016) Triallelic population genomics for inferring correlated fitness effects of same site nonsynonymous mutations. \emph{Genetics} 203:513.}\\
%\item - Introduces a model for the correlated distribution of mutation fitness effects at a genomic locus. To infer the strength of that correlation, we developed a novel diffusion approach to simulating the frequencies of triallelic sites, which we applied to public \emph{D. melanogaster} data. Remarkably, we found that the correlation coefficient we inferred matches that from direct biochemical experiments on bacterial, yeast, and human proteins, suggesting that it is a fundamental property of protein evolution.
%\item - My Ph.D. student developed the method and carried out most of the analysis, which I designed and supervised. We co-wrote the paper.
\ding{73} \href{http://www.genetics.org/content/203/1/NP}{Highlighted by the editors of \emph{Genetics}.}



\item \href{https://doi.org/10.1093/molbev/msx226}{Hsieh P\trainee, Hallmark B, Watkins J, Karafet TM, Osipova LP, \textbf{Gutenkunst RN}\corresponding, Hammer MF\corresponding (2017) Exome sequencing provides evidence of polygenic adaptations and deciphers demographic prehistory in indigenous Siberian populations. \emph{Molecular Biology and Evolution} 34:2913.}\\
\ding{73}  \href{https://www.eurekalert.org/pub_releases/2017-09/mbae-ccf090817.php}{Press release on EurekaAlert.}

\item \href{http://doi.org/10.1534/genetics.116.190611}{Lynch M\corresponding, \textbf{Gutenkunst R}, Ackerman M, Spitze K, Ye Z, Maruki T, Jia Z (2017) Population genomics of \emph{Daphnia pulex}. \emph{Genetics} 206:315.}

\item \href{http://doi.org/10.1111/mec.14131}{Qi X\corresponding, An H, Ragsdale AP\trainee, Hall TE, \textbf{Gutenkunst RN}, Pires JC, Barker MS (2017) Genomic inferences of domestication events are corroborated by written records in \textit{Brassica rapa}. \emph{Molecular Ecology} 206:315.}

\item \href{https://doi.org/10.1534/genetics.117.201251}{Ragsdale AP\trainee\corresponding, \textbf{Gutenkunst RN}\corresponding (2017) Inferring demographic history using two-locus statistics. \emph{Genetics} 206:1037.}

\item \href{https://doi.org/10.1093/bioinformatics/bty010}{Mannakee BK\trainee \corresponding, Balaji U, Witkiewicz AK, \textbf{Gutenkunst RN}\corresponding, Knudsen ES\corresponding (2018)
Sensitive and specific post-call filtering of genetic variants in xenograft and primary tumors. \emph{Bioinformatics} 34:1713.}
	
\item \href{https://doi.org/10.1186/s40246-018-0172-4}{Struck TJ\trainee, Mannakee BK\trainee, \textbf{Gutenkunst RN}\corresponding (2018) The impact of genome-wide association studies on biomedical research publications. \emph{Human Genomics} 12:38.}



\item \href{https://doi.org/10.7554/eLife.54967}{Adrion JK\equal, Cole CB\equal, Dukler N\equal, Galloway JG\equal, Gladstein AL\equal, Gower G\equal, Kyriazis CC\equal, Ragsdale AP\equal, Tsambos G\equal, Baumdicker F, Carlson J, Cartwright RA, Durvasula A, Kim BY, McKenzie P, Messer PW, Noskova E, Vecchyo DO, Racimo F, Struck TJ\trainee, Gravel S, \textbf{Gutenkunst RN}, Lohmeuller KE, Ralph PL, Schrider DR, Siepel A, Kelleher J\corresponding, Kern AD\corresponding (2020) A community-maintained standard library of population genetic models. \emph{eLife} 9:e54967.}

\item \href{https://doi.org/10.1093/molbev/msaa042}{Blischak PD\trainee\corresponding, Barker MS, \textbf{Gutenkunst RN} (2020) Inferring the demographic history of inbred species from genome-wide SNP frequency data. \emph{Molecular Biology and Evolution} 37:2124.}

\item \href{https://doi.org/10.1093/nargab/lqaa004}{Mannakee BK\trainee, \textbf{Gutenkunst RN}\corresponding (2020) BATCAVE: Calling somatic mutations with a tumor- and site-specific prior. \emph{NAR: Genomics and Bioinformatics} 2:lqaa004.}


\item \href{https://doi.org/10.1093/molbev/msab162}{Huang X\trainee, Fortier AL\trainee, Coffman AJ\trainee, Struck TJ\trainee, Irby MN\trainee, James JE\trainee, Le\'on-Burguete JE\trainee, Ragsdale AP\trainee, \textbf{Gutenkunst RN}\corresponding
(2021) Inferring genome-wide correlations of mutation fitness effects between populations. \emph{Molecular Biology and Evolution} 38:4588.}

\item \href{https://doi.org/10.1093/molbev/msaa305}{\textbf{Gutenkunst RN}\corresponding (2021) dadi.CUDA: Accelerating population genetic inference with Graphics Processing Units. \emph{Molecular Biology and Evolution} 38:2177.}

\item \href{https://doi.org/10.1111/1755-0998.13355}{Blischak PD\trainee\corresponding, Barker MS, \textbf{Gutenkunst RN} (2021) Chromosome-scale inference of hybrid speciation and admixture with convolutional neural networks. \emph{Molecular Ecology Resources} 21:2676.}

\item \href{https://doi.org/10.1093/genetics/iyac131}{Gower G\equal, Ragsdale AP\equal, Bisschop G, \textbf{Gutenkunst RN}, Hartfield M, Noskova E, Schiffels S, Struck TJ\trainee, Kelleher J\corresponding, Thornton K (2022) Demes: a standard format for demographic models. \emph{Genetics} 222:iyac131.}

\item \href{https://doi.org/10.1016/j.cell.2022.04.008}{Marchi N\equal, Winkelbach L\equal, Schulz I\equal, Brami M\equal, Hofmanov\'a Z, Bl\"ocher J,
Reyna-Blanco CS, Diekmann Y, Thi\'ery A, Kapopoulou A, Link V, Piuz V, Kreutzer S, Figarska SM, Ganiatsou E, Pukaj A, Struck TJ\trainee, \textbf{Gutenkunst RN}, Karul N, Gerritsen F, Pechtl J, Peters J, Zeeb-Lanz A,
Lenneis E, Teschler-Nicola M, Triantaphyllou S, Stefanovi\'c S, Papageorgopoulou C, Wegmann~D\corresponding, Burger~J\corresponding, Excoffier L\corresponding (2022)
The genomic origins of the world’s first farmers. \emph{Cell} 185:1842.}

\item \href{https://doi.org/10.1111/mec.16391}{Prata KE\corresponding, Riginos C, \textbf{Gutenkunst RN}, Latijnhouwers K, Sánchez JA, Englebert N, Hay K, Bongaerts P (2022) Deep connections: divergence histories with gene flow in mesophotic \emph{Agaricia} corals. \emph{Molecular Ecology} 31:2511.}

\item \href{https://doi.org/10.7554/eLife.74510}{Shaheen MF$^\mathrm{E}$\corresponding, Tse JY$^\mathrm{E}$\corresponding, Sokol ES, Masterson M, Bansal P, Rabinowitz I, Tarleton CA, Dobroff AS, Smith TL, Bocklage TJ, Mannakee BK\trainee, \textbf{Gutenkunst RN}, Bischoff JE, Ness SA, Riedlinger GM, Groisberg R, Pasqualini R, Ganesan S\corresponding, Arap W\corresponding (2022)
Genomic landscape of lymphatic malformations: A case series and response to the PI3K$\alpha$ inhibitor Alpelisib in an N-of-One clinical trial. \emph{eLife} 11:e74510.}

\item \href{https://doi.org/10.1093/genetics/iyad107}{Blischak PD\trainee\corresponding, Sajan M\trainee, Barker MS, \textbf{Gutenkunst RN}\corresponding (2023) Demographic history inference and the polyploid continuum. \emph{Genetics} 224:iyad107.}\\
\ding{73} Featured by the editors of \emph{Genetics}.

\item \href{https://doi.org/10.7554/eLife.84874}{Lauterbur ME\equal, Cavassim MIA\equal, Gladstein AL\equal, Gower G\equal, Pope NS\equal, Tsambos G\equal, Adrion J, Belsare S, Biddanda A, Caudill V, Cury J, Echevarria I, Haller BC, Hasan AR, Huang X, Iasi LNM, Noskova E, Ob{\v s}teter J, Pavinato VAC, Pearson A, Peede D, Perez MF, Rodrigues MF, Smith CCR, Spence JP, Teterina A, Tittes S, Unneberg P, Vazquez JM, Waples RK, Wohns AW, Wong Y, Baumdicker F, Cartwright RA, Gorjanc G, \textbf{Gutenkunst RN}, Kelleher J, Kern AD, Ragsdale AP, Ralph PL, Schrider DR, Gronau I\corresponding (2023) Expanding the stdpopsim species catalog, and lessons learned for realistic genome simulations. \emph{eLife} 12:RP84874.}

\item \href{https://doi.org/10.1086/726010}{Tuffaha M, Varakunan S, Castellano D\trainee, \textbf{Gutenkunst RN}, Wahl LM\corresponding (2023) Shifts in mutation bias promote mutators by altering the distribution of fitness effects. \emph{American Naturalist} 202:503.}

\item \href{https://doi.org/10.1093/molbev/msae077}{Tran LN\trainee, Sun CK\trainee, Struck TJ\trainee, Sajan M\trainee, \textbf{Gutenkunst RN}\corresponding (2024) Computationally efficient demographic history inference from allele frequencies with supervised machine learning. \emph{Molecular Biology and Evolution} 41:msae077.}

\end{enumerate}


\subsection*{Reviews \& Commentaries}
\begin{enumerate}


\item \textbf{Gutenkunst RN} (2002) {Extracting light from water: Sonoluminescence}.
\newblock \emph{Caltech Undergraduate Research Journal} 2:16.

%\begin{hanglist}[0.1in]
%\item 
%- Connects the Girolami and Calderhead's results to sloppiness and to unpublished results from my graduate thesis.
%\item - I co-wrote this invited response. 
%\end{hanglist}

\item \href{http://doi.org/10.1007/978-3-540-37654-5}{Ramachandran S, Tang H, \textbf{Gutenkunst RN}, Bustamante CD (2010) {Genetics and
  genomics of human population structure}.
\newblock In MR~Speicher, AG~Motulsky, SE~Antonarakis, editors, \emph{Vogel and
  Motulsky's Human Genetics: Problems and Approaches}, pages 589--615. Springer
  Verlag, Germany.}
%\begin{hanglist}[0.1in]
%\item 
%- Reviews techniques for analyzing human population-genetic data and what has been learned.
%\item - I wrote the sections on quantitative models of demographic history and selection.
%\end{hanglist}

\item \href{http://doi.org/10.1111/j.1467-9868.2010.00765.x}{\grad Transtrum MK, \textbf{Gutenkunst RN}, Chen Y, Machta BB, Sethna JP
  (2011)
\newblock Discussion of ``{R}iemannian manifold {L}angevin and {H}amiltonian
  {M}onte {C}arlo methods'' by {G}irolami and {C}alderhead.
\newblock \emph{Journal of the Royal Statistical Society: Series B} 73:
  199.}

\item \href{http://doi.org/10.1007/978-3-319-21296-8_11}{Mannakee BK\trainee, Ragsdale AP\trainee, Transtrum MK, \textbf{Gutenkunst RN}\corresponding (2016) Sloppiness and the geometry of parameter space. In D Gomez-Cabrero, L Geris editors, \textit{Uncertainty in Biology: a Computational Modeling Approach}, pages 271--291. Springer International, Switzerland.}
%\begin{hanglist}[0.1 in]
%\item - Reviews the practical implications of sloppiness for modelers in biology.
%\item - I was invited to write this review chapter. I assembled the team of authors, including two of my Ph.D. students, and I supervised the writing and the generation of results from a novel example application.
%\end{hanglist}

\item \href{https://doi.org/10.1093/gbe/evac088}{Johri P\corresponding, Eyre-Walker A, \textbf{Gutenkunst RN}, Lohmueller KE, Jensen JD\corresponding (2022) On the prospect of achieving accurate joint estimation of selection with population history. \emph{Genome Biology and Evolution} 14:evac088.}


\end{enumerate}


\subsection*{Non-refereed Manuscripts}
\begin{enumerate}
\item \href{
https://doi.org/10.48550/arXiv.0712.3240}{\grad \textbf{Gutenkunst RN}, Sethna JP (2007) Adaptive mutation in a geometrical model of chemotype evolution. arXiv.
https://doi.org/10.48550/arXiv.0712.3240}

\item \href{http://www.birs.ca/workshops/2013/13w5080/report13w5080.pdf}{Loewe L, Swain P, \textbf{Gutenkunst R} (2013) Mathematical Tools for Evolutionary Systems Biology. Banff International Research Station 5-day workshop report.\\http://www.birs.ca/workshops/2013/13w5080/report13w5080.pdf}

\item \href{https://doi.org/10.1101/138792}{Wang X\trainee, Li P, \textbf{Gutenkunst RN}\corresponding (2017) Systematic effects of mRNA secondary structure on gene expression and molecular function in budding yeast. bioRxiv. https://doi.org/10.1101/138792}

\end{enumerate}

\subsection*{Software}
\begin{enumerate}

\item \href{http://github.com/GutenkunstLab/SloppyCell}{\grad \textbf{Gutenkunst RN}, Atlas JC, Casey FP, Kuczenski RS, Waterfall JJ, Myers CR, Sethna JP (2007) 
SloppyCell.} \url{http://github.com/GutenkunstLab/SloppyCell}

\item\textbf{Gutenkunst RN} (2009)
\dadi\ -- Diffusion Approximations for Demographic Inference. \\ \url{https://bitbucket.org/gutenkunstlab/dadi}

\item Huang X\trainee, Struck TJ\trainee, Davey S, \textbf{Gutenkunst RN} (2022) dadi-cli.\\ \url{https://github.com/xin-huang/dadi-cli}

\item Tran LN\trainee, Tran LN\trainee, Sun CK\trainee, Struck TJ\trainee, \textbf{Gutenkunst RN} (2023) donni -- Demography Optimization via Neural Network Inference. \url{https://github.com/lntran26/donni}

\end{enumerate}

%\clearpage

\section*{Publications in Progress}
\subsection*{Refereed Journal Articles}
\begin{enumerate}

\item \href{https://doi.org/10.1101/2023.06.15.545182}{Huang X\trainee\equal\corresponding, Struck TJ\trainee\equal, Davey SW, \textbf{Gutenkunst RN}\corresponding (submitted) dadi-cli: Automated and distributed population genetic model inference from allele frequency spectra. Preprint at\\https://doi.org/10.1101/2023.06.15.545182}

\item \href{https://doi.org/10.1101/2024.04.30.591900}{Tuffaha MZ, Castellano D\trainee, Serrano Colom\'e C, \textbf{Gutenkunst RN}, Wahl LM\corresponding (submitted) Non-hypermutator cancers access driver mutations through reversals in germline mutational bias. Preprint at https://doi.org/10.1101/2024.04.30.591900}

\item \href{https://doi.org/10.1101/2022.08.24.505198}{Vourlaki I-T\equal, Castellano D\trainee\equal, \textbf{Gutenkunst RN}\corresponding, Ramos-Onsins SE\corresponding (submitted) Detection of domestication signals through the analysis of the full distribution of fitness effects using forward simulations and polygenic adaptation. Preprint at https://doi.org/10.1101/2022.08.24.505198}

\end{enumerate}

\section*{Conferences/Scholarly Presentations}
% XXX: Current rank
\subsection*{External Departmental Seminars}
\begin{longtable}[l]{l l}
2008 & Department of Computational and Systems Biology, University of Pittsburgh, Pittsburgh, PA\\
2008 & Mathematical Biology Program, University of British Columbia, Vancouver, Canada \\
2008 & Santa Fe Institute, Santa Fe, NM\\
2009 & Centre for Integrative Bioinformatics, Vrije University, Amsterdam, Netherlands\\
2009 & Program in Bioinformatics and Integrative Biology, University of Massachusetts, Worcester, MA\\
2010 & Department of Biology, Boston College, Boston, MA\\
2010 & Department of Physics, Emory University, Atlanta, GA\\
2010 & q-bio Summer School, Los Alamos, NM\\
2010 & BIO5 Institute, University of Arizona\\
2011 & Department of Engineering Sciences and Applied Mathematics, Northwestern University\\
& Chicago, IL\\
2013 & Mathematical Biology Research Program, University of Utah, Salt Lake City, UT\\
2013 & Networks Seminar, University of Houston, Houston, TX\\
2015 & Program in Computational Biology, University of Pittsburgh and Carnegie Mellon University\\
& Pittsburgh, PA\\
2015 & Department of Biology, Temple University, Philadelphia, PA\\
2015 & Center for Computational Biology, University of California, Berkeley, CA\\
2016 & Department of Physics and Astronomy, Brigham Young University, Provo, UT\\
2017 & Center for Bioinformatics Research, Indiana University, Bloomington, IN\\
2017 & Systems Biology Seminar, Boston University, Boston, MA\\
2018 & Bioinformatics Forum, University of Pennsylvania, Philadelphia, PA\\
2018 & Institute for Genomics and Evolutionary Medicine, Temple University, Philadelphia, PA\\
2021 & School of Biological Sciences, Washington State University, Pullman, WA\\
2022 & Department of Biological Sciences, Northern Arizona University, Flagstaff, AZ\\
\end{longtable}


\subsection*{Invited Conference Presentations}
\begin{longtable}[l]{l l}
2009 & Banff International Research Station Workshop: New Mathematical Challenges from Molecular\\
&Biology and Genetics, Banff, Canada\\
2009 & Lorentz workshop: Data Analysis, Parameter Identification and Experimental Design\\
&in Systems Biology, Leiden, Netherlands\\
2012 & Mathematical Biosciences Institute workshop: Robustness in Biological Systems,\\
&Columbus, OH\\
2012 & American Mathematical Society Fall Western Sectional Meeting, Tucson, AZ\\
2014 & Indonesian-American Kavli Frontiers of Science Symposium, Medan, Indonesia\\
2015 & Society for Molecular Biology and Evolution Annual Meeting, Vienna, Austria\\
2016 & Symposium on Cell Signaling, Santa Fe, NM\\
2022 & Software Tools for Open Science, National Institutes of Health, Online\\
2023 & National Association of Biology Teachers Professional Development Conference, Baltimore, MD\\
2024 & Advancing R1 Teaching Faculty for Undergraduate Learning (ARTFUL) Conference, Houston, TX\\
\end{longtable}


%\clearpage

\subsection*{Select Contributed Conference Presentations}
\begin{longtable}[l]{l l}
%%\item[SELECT\\CONTRIBUTED\\PRESENTATIONS]
2005 & Sixth International Conference on Systems Biology, Boston, MA\\
	& \ding{73} Contributed abstract selected for platform presentation in student symposium\\
2007 & Society for Molecular Biology and Evolution Annual Meeting, Halifax, Canada\\
	& \ding{73} Contributed abstract selected for platform presentation\\
2008 & Cornell Postdoc Research Day\\
	& \ding{73} Winner of a Best Poster Presentation award\\
2010 & FASEB Summer Research Conference: Immunoreceptors, Keystone, CO\\
	& \ding{73} Contributed abstract selected for platform presentation\\
2010 & Fourth Annual q-bio Conference on Cellular Information Processing, Santa Fe, NM\\
        & \ding{73} Contributed abstract selected for platform presentation\\
2010 & iEvoBio (Informatics for Phylogenetics, Evolution and Biodiversity) Conference, Portland, OR\\
	& \ding{73} Contributed abstract selected for platform presentation\\
2011 & Society for the Study of Evolution Annual Meeting, Portland, OR\\
2011 & Society for Molecular Biology and Evolution Annual Meeting, Kyoto, Japan\\
        & \ding{73} Contributed abstract selected for platform presentation\\
2011 & Mechanisms of Protein Evolution, Denver, CO\\
2013 & Banff International Research Station Workshop: Mathematical Tools for Evolutionary\\
&Systems Biology, Banff, Canada\\
2013 & Society for Mathematical Biology Annual Meeting, Tempe, AZ\\
2013 & Society for Molecular Biology and Evolution Annual Meeting, Chicago, IL\\
2014 & Society for Molecular Biology and Evolution Annual Meeting, San Juan, PR\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2016 & Biology of Genomes, Cold Spring Harbor, NY\\
2016 & Allied Genetics, Orlando, FL\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2017 & International Society for Evolution, Ecology, and Cancer Annual Meeting, Tempe, AZ\\
2018 & Population, Evolutionary, and Quantitative Genetics, Madison, Wisconsin\\
         & \ding{73} Contributed abstract selected for platform lightning talk presentation\\
2018 & Probabilistic Modeling in Genomics, Cold Spring Harbor, NY\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2019 & Society for Molecular Biology and Evolution Annual Meeting, Manchester, UK\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2020 & The Allied Genetics Conference, Online\\
2022 & Probabilistic Modeling in Genomics, Online\\
2022 & Population, Evolutionary, and Quantitative Genetics, Pacific Grove, CA\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2022 & Society for Molecular Biology and Evolution Everywhere, Online\\
         & \ding{73} Contributed abstract selected for platform presentation\\
2023 & Probabilistic Modeling in Genomics, Cold Spring Harbor, NY\\
2023 & Society for the Study of Evolution Annual Meeting, Albuquerque, NM\\
2024 & The Allied Genetics Conference, Washington, DC\\
\end{longtable}

%\section*{Individual Student Contact}
%\underline{Graduate students with degrees from my lab}\\
%\begin{longtable}[l]{l L}
%PingHsun Hsieh & Ecology and Evolutionary Biology, Ph.D.\\
%                        & (co-advised with Prof.\ Michael Hammer)\\
%                        & Graduated 2016, now postdoc with \href{https://eichlerlab.gs.washington.edu/}{Prof.\ Evan Eichler at the University of Washington}\\
%                        & \begin{tabular}{ll}
%                        \underline{Awards}\\
% 2013 & Charles J. Epstein Trainee Award for Excellence in Human \\
%& Genetics Research (one time: \$750)\\
%                        2013 & Galileo Circle Scholarship (one-time: \$1,000)\\
%                        2013 & Government of Taiwan Study Abroad Scholarship\\
%                             & (one year: $\approx$\$10,000, renewed in 2014)\\
%                        \end{tabular}\\
%Travis Stuck & Molecular \& Cellular Biology, M.S.\\
%                       & Graduated 2016, continuing to work in my lab\\
%
%\end{longtable}
%
%\underline{Current graduate students in my lab}\\
%\begin{longtable}[l]{l L}
%                        Aaron Ragsdale & Applied Mathematics, Ph.D.\\
%               & \begin{tabular}{ll}
%               \underline{Awards}\\      
%               2012 & NSF IGERT Fellowship in Comparative Genomics\\
%                    & (one year: tuition + \$30,000 stipend)\\
%               2015 & H.E. Carter Travel Award (one time: \$600)\\
%                                                      \end{tabular}\\
%%\end{longtable}\\
%%\begin{tabulary}{\textwidth}{l L}
%Brian Mannakee & Biostatistics, Ph.D\\
%				& \begin{tabular}{ll}
%               \underline{Awards}\\      
%              2013 & Galileo Circle Scholarship (one-time: \$1,000)\\
%              2013 & ARCS Foundation Scholarship \\
%                   & (one year: tuition + \$9,000 per year, renewed in 2014)\\
%              2013 & NSF Graduate Research Fellowship\\
%                   & (three years, deferred until 2015: tuition + \$30,000 per year stipend)\\
%              \end{tabular}\\
%\end{longtable}

\section*{Grants and Contracts}
% (list percent effort on grant; role [PI, co-PI]; all co-PIs or co-Is; source and amount
\subsection*{Current Federal Research Support}
\begin{longtable}[l]{l l}
2/12/19--1/31/24 & \href{https://projectreporter.nih.gov/project_info_description.cfm?aid=9659095}{NIH R01 GM127348: \$1,472,436} (No-cost extension to 1/31/25)\\
                  & \href{https://projectreporter.nih.gov/project_info_description.cfm?aid=9659095}{PI: Ryan Gutenkunst}\\
                  &  \href{https://projectreporter.nih.gov/project_info_description.cfm?aid=9659095}{Joint Inferences of Natural Selection Between Sites and Populations}\\
                  & Role: PI (16.6\% effort)\\
                  & Additional administrative supplement of \$199,665 awarded in 2020  from NIH Office\\
                  & \qquad of Data Science Strategy to support work with Nirav Merchant to bring cloud\\
                  & \qquad computing to dadi\\
5/1/23--2/29/28 & \href{https://reporter.nih.gov/project-details/10623079}{NIH R35 GM149235: \$1,667,719}\\
                  & \href{https://reporter.nih.gov/project-details/10623079}{PI: Ryan Gutenkunst}\\
                  &  \href{https://reporter.nih.gov/project-details/10623079}{Genomic Inferences of History and Selection across Populations and Time}\\
                  & Role: PI (29.8\% effort)\\
\end{longtable}


\subsection*{Past Federal Research Support}
\begin{longtable}[l]{l l}
3/1/12--2/29/16 & \href{http://www.nsf.gov/awardsearch/showAward?AWD_ID=1146074}{NSF DEB-1146074: \$551,964}\\
                  & \href{http://www.nsf.gov/awardsearch/showAward?AWD_ID=1146074}{PI: Ryan Gutenkunst}\\
                  & \href{http://www.nsf.gov/awardsearch/showAward?AWD_ID=1146074}{Demographic History Inference from Genomic Linkage and Allele Frequency Spectra}\\
                  & Role: PI (16.6\% effort)\\
9/01/14--1/31/18 & DARPA WF911NF-14-1-0395: \$3,630,769\\
                  & PI: Mihai Surdeanu; Co-Is: Kobus Barnard, Angus Forbes, Ryan Gutenkunst,\\
                  & Clayton Morrison, Guang Yao\\
                  & REACH: Reading and Assembling Contextual and Holistic Big Mechanisms from Text\\
                  & Role: Co-PI (8.3\% effort)\\
9/01/16--8/31/19 & \href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1625015}{NSF DUE-1625015: \$598,690}\\
                  & \href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1625015}{PI: Molly Bolger; Co-PIs: Lisa Elfring, Jennifer Katcher}\\
                  &  \href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1625015}{Authentic Scientific Practices in the Classroom: a Model-Based-Inquiry Curriculum}\\
                  &\href{https://www.nsf.gov/awardsearch/showAward?AWD_ID=1625015}{for the Introductory Biology Laboratory}\\
                  & Role: Senior personnel (4.1\% effort)\\
\end{longtable}

                  
%\subsection*{Pending Federal Research Support}
%\begin{longtable}[l]{l l}
%2/1/17-- 1/31/22 & NSF DEB: \$853,656\\\
%                  & PI: Ryan Gutenkunst\\
%       		& CAREER: Inferring natural selection on protein networks\\
%7/1/2017 -- 6/30/2022 & NIH R01: \$1,949,897\\
%		& PIs: Michael Blinov, Kevin Brown, and Ryan Gutenkunst\\
%		& Accounting for parameter and model uncertainty in VCell using Sloppy\\
%		& Models theory\\
%\end{longtable}
%\addtocounter{footnote}{-1}
%\footnotetext{In the 2015 cycle, this proposal was one of 33 rated High Priority by the Evolutionary Genetics panel, but it was not among the $\sim$46 proposals funded. I was told by my Program Officer that my proposal was unfunded, in part, because my Co-PI contract from DARPA meant that I needed the funds less than other applicants.}
%\addtocounter{footnote}{+1}
%\footnotetext{Will be resubmitted for the November 5, 2016 deadline. The last submission received a percentile score of 35.}


\subsection*{Past Institutional Research Support}
\begin{longtable}[l]{l l}
6/1/15--5/31/16 & University of Arizona Center for Insect Science: \$10,000\\
& PI: Anna Dornhaus; Co-Is: Ryan Gutenkunst, Gavin Leighton\\
& Testing the genetic toolkit of social behavior hypothesis using detailed descriptions\\
&of behavior and RNA-sequencing experiments in Temnothorax rugatulus\\
\end{longtable}

\section*{Extent of Teaching}
\subsection*{List of Courses Taught}
\begin{longtable}[l]{l l r r}
2010, Fall & Cell Systems, MCB 572A & 8 students, 3 units & 25\%\\
2011, Fall & Cell Systems, MCB 572A & 28 students, 3 units & 33\%\\
2012, Spring & Key Concepts in Quantitative Biology, MCB 315 & 15 students, 4 units & 60\%\\
2012, Fall & Key Concepts in Quantitative Biology, MCB 315 & 13 students, 4 units & 80\%\\
2012, Fall & Cell Systems, MCB 572A & 18 students, 4 units & 50\%\\
2013, Fall & Key Concepts in Quantitative Biology, MCB 315 & 12 students, 4 units & 100\%\\
2014, Fall & Cell Systems, MCB 572A & 18 students, 4 units & 33\%\\
2014, Fall & Introductory Biology I, MCB 181 & 336 students, 3 units & 15\%\\
2014, Fall & Key Concepts in Quantitative Biology, MCB 315 & 15 students, 4 units & 100\%\\
2015, Fall & Introductory Biology I, MCB 181 & 350 students, 3 units & 15\%\\
2015, Fall & Key Concepts in Quantitative Biology, MCB 315 & 11 students, 4 units & 100\%\\
2015, Fall & Cell Systems, MCB 572A & 19 students, 4 units & 33\%\\
2016, Fall & Key Concepts in Quantitative Biology, MCB 315 & 8 students, 4 units & 100\%\\
2016, Fall & Cell Systems, MCB 572A & 13 students, 4 units & 50\%\\
2018, Fall & Genomic Medicine Colloquium, MCB 195B & 18 students, 1 unit& 100\%\\
2018, Fall & Quantitative Biology, MCB 315 & 13 students, 3 units & 100\%\\
2018, Fall & Scientific Communication, MCB 575 & 7 students, 3 units & 50\%\\
2019, Fall & Big Data in Molecular Biology and Biomedicine, MCB 447 & 7 students, 3 units & 100\%\\
2020, Fall & Genomic Medicine Colloquium, MCB 195B & 15 students, 1 unit& 100\%\\
2020, Fall & Quantitative Biology, MCB 315 & 19 students, 3 units & 67\%\\
2020, Fall & Scientific Communication, MCB 575 & 10 students, 3 units & 50\%\\
2021, Fall & Genomic Medicine Colloquium, MCB 195B & 18 students, 1 unit& 100\%\\
2021, Fall & Big Data in Molecular Biology and Biomedicine, MCB 447/547 & 16 students, 3 units & 50\%\\
2021, Fall & Scientific Communication, MCB 575 & 5 students, 3 units & 50\%\\
2022, Fall & Genomic Medicine Colloquium, MCB 195B & 20 students, 1 unit& 100\%\\
2022, Fall & Quantitative Biology, MCB 315 & 22 students, 3 units & 50\%\\
2022, Fall & Scientific Communication, MCB 575 & 10 students, 3 units & 50\%\\
2023, Spring & Genetics, Ancestry, and Race, MCB 295E & 25 students, 1 unit & 100\%\\
2023, Fall & Genetics, Ancestry, and Race, MCB 295E & 39 students, 1 unit & 100\%\\
2023, Fall & Big Data in Molecular Biology and Biomedicine, MCB 447/547 & 19 students, 3 units & 33\%\\
\end{longtable}

\subsection*{Guest Lectures}
\begin{longtable}[l]{l l}
2010 & Quantitative Biology colloquium, MATH 596A (4 sessions)\\
2010, 2013 & Research Topics in Computer Science, CSC 296H/496H\\
2011 & Introduction to Biophysics, PHYS 430/530\\
2011 & Genetic and Molecular Networks, MCB 546\\
2011 & Recent Advances in Genetics, GENE 670\\
2011, 2013, 2014 & Functional and Evolutionary Genomics, ECOL 453/553\\
%2011 & Mentor for Honors Introductory Biology, MCB 181H (5 sessions)\\
%2012 & Cell Systems, MCB 572\\
2013 & Complex Systems: Networks \& Self-organization in Biology, ECOL 496H/596H\\
2014, 2015, 2016 & Initiative for Maximizing Student Development colloquium, MCB 595E\\
2014, 2016 & Introduction to Modeling in Biology, ECOL 519\\
2015 & Bioinformatics, ECOL 346\\
2016 & Seminar in Bioinformatics, ECOL 296B\\
2022 & Race, Ethnicity, and the American Dream, ANTH 150\\
\end{longtable}

\section*{Individual Trainee Contact}

\subsection*{Collaborations with Undergraduates on Research Projects}
\begin{longtable}[l]{l l l}
Travis Woodrow & Fall 2012--Spring 2013 & Computer Science\\
Michael Iuzzolino & Summer 2013 & Mathematics\\
Jose Leon-Burguete & Winter 2017 & Genomic Sciences, National Autonomous\\ 
                   &             & University of Mexico (J-1 Student Intern)\\
Setayesh Odmidian & Spring 2019--Summer 2019 & Biology\\
Mathews Sajan & Fall 2019--Spring 2023 & Pre-Health and Business\\
Connie Sun & Summer 2020--Spring 2023 & Ecology \& Evolutionary Biology and Computer Science\\
Oliva Fernflores & Spring 2022--present & Bioinformatics\\
Lilith Kotler & Fall 2023--present & Molecular \& Cellular Biology\\
\end{longtable}

\subsection*{Collaborations with Graduate Students on Research Projects (Lab Rotations)}
\begin{longtable}[l]{l l l}
PingHsun Hsieh & Fall 2010 & Ecology \& Evolutionary Biology\\
Mary Paniscus & Fall 2010 & Genetics\\
Aaron Ragsdale & Fall 2011 & Applied Mathematics\\
Liang Wu & Fall 2012 & Ecology \& Evolutionary Biology\\
Adam Grant & Fall 2016 & Arizona Biological \& Biomedical Sciences\\
Amber Koslucher & Summer 2018 & Biostatistics\\
Hao Zhang & Spring 2019 & Arizona Biological \& Biomedical Sciences\\
Theodore Meissner & Fall 2021 & Applied Mathematics\\
Yanghuan Yu & Fall 2021 & Arizona Biological \& Biomedical Sciences\\
Stephen Cooke & Spring 2023 & Ecology \& Evolutionary Biology\\
Raymond Hon & Spring 2024 & Arizona Biological \& Biomedical Sciences\\
Heng Wu & Spring 2024 & Applied Mathematics\\
\end{longtable}

%\subsection*{Mentoring and Career Counseling}
%I mentor graduate students in the Initiative for Maximizing Student Development program on applying for National Science Foundation Graduate Research Fellowships.

\subsection*{Theses Directed and In Progress}
\begin{longtable}[l]{l l l}
Brian Mannakee & Spring 2012 & B.S. in Biochemistry\\
 & & \href{http://hdl.handle.net/10150/244452}{``Evolutionary rate at the protein domain level is constrained by}\\
&& \href{http://hdl.handle.net/10150/244452}{importance to network dynamics''}\\
Katherine Cunningham & Spring 2013 & B.S. in Molecular \& Cellular Biology and Computer Science\\
 && ``Optimization in the demographic simulation software  \dadi''\\
 Siddharth Pandya & Spring 2013 & B.S. in Biochemistry\\
 && \href{http://hdl.handle.net/10150/297727}{``Directional selection on tyrosine frequencies in eukaryotes}\\
 &&\href{http://hdl.handle.net/10150/297727}{versus solvent accessibility''}\\
 Travis Struck & Spring 2016 & M.S. in Molecular \& Cellular Biology\\
                    & & ``Research effort and evolutionary properties of genes''\\
Brandon Jernigan & Spring 2017 & B.S. in Chemical Engineering\\
& & ``Evolutionary rate covariation of domain families''\\
Alyssa Fortier & Spring 2018 & B.S. in Molecular \& Cellular Biology and Mathematics\\
&& \href{http://hdl.handle.net/10150/630373}{``Improving the robustness of dominance and selection inference''}\\
&& Awarded a NSF Graduate Research Fellowship\\
Megan Irby & Spring 2020 & B.S. in Molecular \& Cellular Biology and Mathematics\\
&& \href{https://repository.arizona.edu/handle/10150/651036}{``The joint distribution of fitness effects of wild tomatoes and a brief}\\
&& \href{https://repository.arizona.edu/handle/10150/651036}{introduction to linkage in DFE inference''}\\
Amy Fan & Spring 2023 & B.S. in Molecular \& Cellular Biology and Statistics \& Data Science\\
&&``Modeling selection bias on recombination rates inferred through\\
&& linkage disequilibirum''
\end{longtable}

\subsection*{Dissertations Directed and In Progress}
\begin{longtable}[l]{l l l}
PingHsun Hsieh & Spring 2016 & Ph.D. in Ecology and Evolutionary Biology\\
 & & (Co-advised with Prof.\ Michael Hammer)\\
 & & \href{http://hdl.handle.net/10150/612540}{``Model-based population genetics in indigenous humans:}\\
 &&\href{http://hdl.handle.net/10150/612540}{Inferences of demographic history, adaptive selection, and African}\\
 &&\href{http://hdl.handle.net/10150/612540}{archaic admixture using whole-genome/exome sequencing data''}\\
 && Left to become postdoc with \href{https://eichlerlab.gs.washington.edu/}{Prof.\ Evan Eichler at the University of Washington}.\\
 && Now \href{https://hsiehph.github.io/}{faculty in Genetics, Cell Biology, and Development}\\
 && \href{https://hsiehph.github.io/}{at the University of Minnesota}.\\
Aaron Ragsdale & Fall 2017 & Ph.D. in Applied Mathematics\\
& & \href{http://hdl.handle.net/10150/622993}{``Multi-allele population genomics for inference of demography}\\
&&\href{http://hdl.handle.net/10150/622993}{and natural selection''}\\
&& Left to become postdoc with \href{http://simongravel.lab.mcgill.ca/}{Prof.\ Simon Gravel at McGill University}.\\
&&Now \href{https://apragsdale.github.io/}{faculty in Integrative Biology at University of Wisconsin—Madison}.\\
Brian Mannakee & Fall 2019 & Ph.D. in Biostatistics\\
& &\href{http://hdl.handle.net/10150/637708}{``Statistical methods for improving low frequency}\\
&&\href{http://hdl.handle.net/10150/637708}{variant calling in cancer genomics''}\\
&&Awarded an NSF Graduate Research Fellowship\\
&&Left to become Bioinformatics Scientist at \href{https://www.foundationmedicine.com}{Foundation Medicine}.\\ 
Linh Tran & In progess & Ph.D. in Genetics\\
 \end{longtable}

\subsection*{Service on Other Dissertation and Graduate Committees}
\begin{longtable}[l]{l l l}
Julio Cesar Ignacio Espinoza & Winter 2015 & Ph.D. in Molecular \& Cellular Biology\\
Cristina Howard & Spring 2015 & Ph.D. in Molecular \& Cellular Biology\\
Ryan Pace & Summer 2015 & Ph.D. in Molecular \& Cellular Biology\\
Dhruv Vig & Summer 2015 & Ph.D. in Molecular \& Cellular Biology\\
August Woerner & Summer 2016 & Ph.D. in Genetics\\
Peter Vinton & Summer 2016 & Ph.D. in Molecular \& Cellular Biology\\
Consuelo Quinto Cort\'{e}s & Fall 2016 & Ph.D. in Genetics\\
Grant Schissler & Spring 2017 & Ph.D. in Statistics\\
Miao Zhang & Spring 2018 & Ph.D. in Statistics\\
%Liang Wu & Ecology \& Evolutionary Biology, Ph.D.\\
%Todd Jarvis & Genetics, M.S.\\
%Samantha Anderson & In progress & Ph.D. in Ecology \& Evolutionary Biology\\
Nicholas Kappler & Spring 2018 & Ph.D. in Applied Mathematics\\
Ariella Gladstein & Summer 2018 & Ph.D. in Ecology \& Evolutionary Biology\\
Nicholas Helle & Summer 2018 & P.S.M. in Applied Biosciences\\
Arron Sullivan & Fall 2018 & Ph.D. in Molecular \& Cellular Biology\\
David Jones & Spring 2019 & P.S.M. in Applied Biosciences\\
%Pearl Lam & Spring 2019 & M.S. in Molecular \& Cellular Biology\\
David Waid & Spring 2019 & P.S.M. in Applied Biosciences\\
Kun Xiong & Spring 2019 & Ph.D. in Molecular \& Cellular Biology\\
Luke Kosinski & Summer 2020 & Ph.D. in Molecular \& Cellular Biology\\
Robert Porter & Summer 2020 & P.S.M. in Applied Biosciences\\
Robert Betterton & Fall 2020 & P.S.M. in Applied Biosciences\\
Minhao Chen & Spring 2021 & P.S.M. in Applied Biosciences\\
Quinea Lassiter & Spring 2021 & P.S.M. in Applied Biosciences\\
Elizabeth Ogunbunmi & Spring 2021 & P.S.M. in Applied Biosciences\\
Cathryn Sephus & Spring 2021 & M.S. in Molecular \& Cellular Biology\\
Christopher Carnahan & Summer 2021 & P.S.M. in Applied Biosciences\\
Adam Grant & Summer 2021 & Ph.D. in Cancer Biology\\
Nicole Walker & Spring 2022 & P.S.M. in Applied Biosciences\\
Chenlu Di & Fall 2022 & Ph.D. in Ecology \& Evolutionary Biology\\
Ondrej Cernicky & Spring 2023 & M.S. in Molecular \& Cellular Biology\\
Joshua Hack & Spring 2023 & M.S. in Molecular \& Cellular Biology\\
Annalisa Medina & Spring 2023 & P.S.M. in Applied Biosciences\\ % Finishing Fall 2022
Alexandra Sundman & Spring 2023 & M.S. in Molecular \& Cellular Biology\\ % Finishing Spring 2023
Shane Thomas & Spring 2023 & Ph.D. in MCB with Emphasis in Science Education\\ % ???
Alhan Yazdi & Spring 2023 & P.S.M. in Applied Biosciences\\
Heather Connick & Winter 2023 & P.S.M. in Applied Biosciences\\
Matt Miller & Spring 2024 & M.S. in Molecular \& Cellular Biology\\ % ???
%Chloe Brindley & In progress & P.S.M. in Applied Biosciences\\
Ulises Hernandez & In progress & Ph.D. in Ecology \& Evolutionary Biology\\
Qiuyu Jiang & In progress & Ph.D. in Ecology \& Evolutionary Biology\\
%Aaron Law & In progress & Ph.D. in Electrical \& Computer Engineering\\ % Couldn't pass Comp Exam with me on committee...
Mary Reed-Weston & In progress & Ph.D. in Genetics\\
Genavieve Sandoval & In progress & Ph.D. in Ecology \& Evolutionary Biology\\
Sawsan Wehebi & In progress & Ph.D. in Genetics\\
Andrew Wheeler & In progress & Ph.D. in Genetics\\
Jiawen Yang & In progress & Ph.D. in Cancer Biology\\
\end{longtable}

\subsection*{Postdoctoral Scholars Trained}
\begin{longtable}[l]{l l}
Alexandra Jilkine & July 2011--July 2013\\
                  & Left to become \href{https://acms.nd.edu/people/alexandra-jilkine/}{Assistant Professor of Applied Mathematics and Computational}\\
                  &\href{https://acms.nd.edu/people/alexandra-jilkine/}{Mathematics and Statistics} at the University of Notre Dame\\
Xia Wang & January 2015--January 2018\\
		& (co-advised with Prof. Guang Yao) Left to become postdoc with Prof. Helen Zhang at\\ &the University of Arizona.\\
Paul Blischak & September 2018--December 2020\\
		& (co-advised with Prof. Mike Barker and supported by an NSF Plant Genome fellowship)\\
		& Left to become Data Scientist at Bayer Crop Science\\
Xin Huang & November 2019--June 2021\\
                & Left to become a postdoc with Martin Kuhlwilm at the University of Vienna\\
Jennifer James & August 2020--April 2021\\
		& Left with a Wenner-Gren Post-PhD Fellowship to work with Martin Lascoux\\
		& at Uppsala University\\
David Castellano & December 2021--present\\
Emanuel Fonseca & May 2022--present\\
Justin Conover & March 2023--present\\
& (co-advised with Prof. Mike Barker and supported by an NSF Plant Genome fellowship)\\
\end{longtable}

\section*{Contributions to Instructional Innovations and Collaborations}
\subsection*{Teaching Workshops Delivered}
\begin{longtable}[l]{l l}
2009--2017 & q-bio Summer School, Albuquerque, NM\\
2016 & Workshop on Population and Speciation Genomics, \v{C}esk\'{y} Krumlov, Czech Republic
\end{longtable}

%\subsection*{Development of online and other course materials}
%I developed all materials for the course MCB 315: Key Concepts in Quantitative Biology.\\
%\hspace*{0.25in}I am developing an instructional module to guide students in the curation of systems biology models from the BioModels database. I am testing the module in my courses MCB 315 and MCB 572 and at the q-bio Summer School. When refined, I intend to disseminate the module widely, so that it may be incorporated into other quantitative biology curricula and engage many students in contributing to the database.

\subsection*{Collaborations on Curricular Committees}
\begin{longtable}[l]{l l}
2011--2017 & Member, Astrobiology undergraduate and graduate minor executive committee\\
2012--present& Member, Molecular \& Cellular Biology undergraduate curriculum committee\\
2013--present & Member, Bioinformatics undergraduate major steering committee\\
2015 & Member, Introductory Physics I redesign committee\\
2020--present & Member, Master's in Genetics curriculum committee
\end{longtable}

\subsection*{Curriculum Development}
I developed the curricula for the MCB 315: Qualitative Biology and for MCB 447/547: Big Data in Molecular Biology and Biomedicine. I also developed a two-week computational cancer lab for the AIM-Bio reformed MCB 181L: Introductory Biology Laboratory. In addition, I developed two 1-unit colloquia courses: MCB 195B: Genomic Medicine and MCB 295E: Genetics, Ancestry, and Race.

\section*{Teaching Awards}
\subsection*{Department and College}
\begin{longtable}[l]{l l}
2013 & Distinguished Early-Career Teaching Award, College of Science
\end{longtable}

% Spring 2023: NRMN (National Research Mentoring Network)Culturally Aware Mentorship training.	
                 
\end{document}
